src/main/java/io/cdap/plugin/gcp/bigquery/connector/BigQueryConnector.java

```diff
@@ -62,8 +69,10 @@
   "tables. BigQuery is Google's serverless, highly scalable, enterprise data warehouse.")
 public final class BigQueryConnector implements DirectConnector {
   public static final String NAME = "BigQuery";
-  public static final String ENTITY_TYPE_DATASET = "dataset";
-  public static final String ENTITY_TYPE_TABLE = "table";
+  public static final String ENTITY_TYPE_DATASET = "DATASET";
+  public static final String ENTITY_TYPE_TABLE = TableDefinition.Type.TABLE.name();
+  public static final String ENTITY_TYPE_VIEW = TableDefinition.Type.VIEW.name();
```

Comment:
I don't see where this is used. I remember we plan to add this type to the `BrowseEntityProperty`?

Reply:
I see this type is already added in the `getTableDetail` method, we can just remove these

Reply:
Here I'd like to expose all the possible `type` values because the `type` in BrowseEntityProperty is a string.

Reply:
If they are not used just remove them since this is dependent on the client, if bigquery decides to add more types in the future, it won't be reflected here, I think `definition.getType()` is enough in `getTableDetail`

Reply:
makes sense.  done :)